docs/ca/dev/preparation-for-going-live.md

```diff
@@ -23,191 +23,108 @@ We've divided the preparation into approximate timeframes, and you can adjust th
+| APP-13 | Application                    | Implement performance guidelines                     | Implement [performance guidelines](/docs/dg/dev/guidelines/performance-guidelines/performance-guidelines.html).                                                                                                                                                                                                              
+| APP-14 | Application                    | Implement Jenkins guidelines                                 | Implement [Jenkins operational best practices](/docs/ca/dev/best-practices/jenkins-operational-best-practices.html).                                                                                                                                                                                                                                          
+| APP-15 | Application                    | Implement publish and sync guidelines                        | Implement applicable [Publish and Sync stability best practices](/docs/ca/dev/best-practices/best-practises-jenkins-stability.html).                                                                                                                               
+| APP-16 | Application                    | Implement general security guidelines                        | Apply [security guidelines](/docs/dg/dev/guidelines/security-guidelines.html).                                                                                                                                                                                                                                                                       |
+| APP-17 | Application                    | Implement password security guidelines                       | Make sure that you don't have any plain-text passwords, private keys, or API secrets in config files or Git repositories.                                                                                                                                                                                                                                    
+| APP-18 | Application                    | Implement credentials security guidelines                    | Minimize the use of personal credentials and choose work-specific accounts based on each environment: production, staging, testing. We highly recommend employing Centralized Credential Management to securely store and manage these credentials.                                                                                                         
```

Comment:
Do you mean that there should be environment specific accounts? I don't get the 'choosing' part